src/DataCollection.UWP/MainPage.xaml

```diff
@@ -0,0 +1,76 @@
+﻿<Page
+    x:Class="Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    xmlns:utils="using:Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.Utils"
+    xmlns:converters="using:Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.Converters"
+    RequestedTheme="Dark"
```

Comment:
I recommend removing the theme request and using the user's Theme. I tested with Light and Dark on my PC and both worked quite well - only change needed was removing explicit text color on the title.